client/lib/analytics/super-props.js

```diff
@@ -23,9 +24,17 @@ const getSuperProps = ( reduxStore ) => ( eventProperties ) => {
 	}
 
 	const path = eventProperties.path ?? getCurrentRoute( state );
-
-	const omitSelectedSite = ! eventProperties.force_site_id && shouldReportOmitBlogId( path );
-	const selectedSite = omitSelectedSite ? null : getSelectedSite( state );
+	const isStepper = path.startsWith( '/setup' );
+	let selectedSite;
+
+	if ( isStepper ) {
```

Comment:
I guess we can handle this inside the `shouldReportOmitBlogId`. WDYT?

Reply:
Sounds good!